src/_reducers/TickReducer.js

```diff
@@ -56,7 +49,7 @@ export default (state = initialState, action) => {
                 epoch: +tick.epoch,
                 quote: +tick.quote,
             };
-            return state.update(symbol, List.of(), v => v.takeLast(1000).push(fromJS(newTick)));
+            return state.update(symbol, List.of(), v => v.takeLast(10000).push(fromJS(newTick)));
```

Comment:
depends on what you mean by correct ;)

for ticks, I hit 10000 easily, so might raise it or we discuss how to handle